packages/manager/modules/nutanix/src/dashboard/controller.js

```diff
@@ -0,0 +1,10 @@
+export default class NutanixCtrl {
+  /* @ngInject */
+  constructor($translate) {
+    this.$translate = $translate;
+  }
+
+  $onInit() {
+    console.log('Hello', this.$translate.instant('world'));
```

Comment:
must we kept that console @radireddy  ?

Reply:
this is auto generated framework code, all these will be deleted or taken care during implementation.